sale_automatic_workflow/models/stock_picking.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2011 Akretion Sébastien BEAU <sebastien.beau@akretion.com>
+# Copyright 2013 Camptocamp SA (author: Guewen Baconnier)
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.tools import float_compare
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    workflow_process_id = fields.Many2one(
+        comodel_name="sale.workflow.process", string="Sale Workflow Process"
+    )
+
+    @api.multi
+    def validate_picking(self):
```

Comment:
```suggestion
    def validate_picking(self):
    # set quantities aromatically and validate picking
```

Reply:
probably picking validation should be optional, and setup as workflow option, if customer have complex road or using serial numbers it could mess up maybe @gdgellatly faced with this problems
@sebalix if you will go for it, make as separate commit to easily back-port it

Reply:
aromatically? :rofl:

Reply:
added as docstring, it smells better :)